etc/p3-config/p3-config.yml

```diff
@@ -219,11 +219,13 @@ p3_users:
 p3_admin_roles:
   - admin
   - heat_stack_owner
+  - monasca-user
 
-# List of roles to apply to admin users in the p3 project.
+# List of roles to apply to regular users in the p3 project.
 p3_user_roles:
   - heat_stack_owner
   - observer
+  - monasca-read-only-user
```

Comment:
Just wondering - who's responsible for creating this role? Are we relying on the ```monasca_identity.sh``` script from the monasca-deploy project?

Reply:
Good question, I guess I am. Ideally its just relying on kayobe to put everything in place, but generally the script assumes the roles are already created AFAIK.

Reply:
It ought to be created as part of this repo, but that's another PR I think.  It will need doing to prevent ordering problems with running p3-config vs running the Monasca deploy playbooks.

Reply:
It seems like it should be the same as heat though?

Reply:
Turns out kolla-ansible does the same for heat: https://github.com/openstack/kolla-ansible/blob/master/ansible/roles/heat/tasks/register.yml#L41.